static/docs/user-guide/running-dvc-on-windows.md

```diff
@@ -47,3 +47,20 @@ The performance of NTFS degrades while handling large volumes of files in a
 directory.
 [Here](https://stackoverflow.com/questions/197162/ntfs-performance-and-large-volumes-of-files-and-directories)
 is the resource for reference.
+
+## Enabling paging with `less`
+
+By default DVC uses `less` to page output for some commands. Although, many
+operating systems have `less` preinstalled, there are still platforms where
+`less` is not installed. This usually happens for Microsoft Windows.
+Fortunately, there is a easy way of installing `less` via
+[Chocolatey](https://chocolatey.org/).
+
+```
```

Comment:
`$ choco ...` + add `dvc` syntax highlighter

Reply:
You sure?

Personally, I don't like when `$` is included, because it's harder to copy :)

Reply:
$ won't be included - our engine automatically takes care of it :)

Reply:
just checked, `$` is not selected - awesome :+1:  
also added to https://github.com/iterative/dvc.org/pull/831/files#diff-20c0689b083dedb8ffeb8a05226dd834R77

thanks!